types.go

```diff
@@ -95,6 +95,11 @@ func (seg *Segments) UnmarshalJSON(b []byte) (err error) {
 	return
 }
 
+func (seg Segments) MarshalJSON() ([]byte, error) {
+	s := fmt.Sprintf("%d", seg)
```

Comment:
There's a lot of overhead for `fmt.Sprintf(` to parse the format string, since we are dealing with an int we should just call `strconv`, ideally directly to a `[]byte`. 

I'm also confused why we can't just marshal the integer directly, ie json.Marshal(int(seg))

Reply:
Same style of comment applies for the rest of the changes in this file.

Reply:
We can't marshal the integer directly because Segments, NumMedia and TwilioDuration are represented in JSON as strings, i.e. we need 123 segments to be marshalled as "123" instead of 123.

```
func (n NumMedia) MarshalJSON() ([]byte, error) {
	s := strconv.AppendUint(nil, uint64(n), 10)
	return json.Marshal(string(s))
}
```

I've changed the MarshalJSON functions to look like the above, we could go one step further and eliminate the json.Marshal entirely by doing something like:

```
func (n NumMedia) MarshalJSON() ([]byte, error) {
	s := make([]byte, 0)
	s = append(s, '"')
	s = strconv.AppendUint(s, uint64(n), 10)
	s = append(s, '"')
	return s, nil
}
```

What do you think?

Reply:
FWIW, I initially found myself agreeing with you. However, after actually making the changes, I feel that fmt.Sprintf best expresses the intention behind the code, i.e. that when Segments is marshalled into JSON, is represented as a JSON string.

Given that this bit of code will only ever be used for tests, I kinda prefer the more succinct but less performant fmt.Sprintf.

Don't feel that strongly about this, happy to go with whatever you think is best. Thanks for the library!